NOTICE

```diff
@@ -0,0 +1,14 @@
+Dotty (https://dotty.epfl.ch)
+Copyright 2012-2019 EPFL
+Copyright 2012-2019 Lightbend, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License"):
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+This software includes projects with other licenses -- see `doc/AUTHORS.md`.
```

Comment:
I think we should just remove AUTHORS.md and add the relevant information in this file directly.

Reply:
Fixed by 68450b6.